model_compression_toolkit/target_platform_capabilities/target_platform/targetplatform2framework/attach2fw.py

```diff
@@ -0,0 +1,47 @@
+from typing import Dict, Tuple, List, Any, Optional
+
+from model_compression_toolkit import DefaultDict
+from model_compression_toolkit.target_platform_capabilities.schema.mct_current_schema import TargetPlatformModel
+from model_compression_toolkit.target_platform_capabilities.target_platform import TargetPlatformCapabilities, \
+    OperationsSetToLayers
+
+
+class TpcAttach2Fw:
+
+    def __init__(self):
+        self._opset2layer = None
+
```

Comment:
why not get it as an input and eliminate the context manager functionality? then you also document it with type hint and documentation

Reply:
As I said, I prefer not to deal with the TPC context manager elimination as part of this refactor